File: Leetcode_Java/String/LengthOfLastWord.java
package Leetcode_Java.String;

public class LengthOfLastWord {

    public int lengthOfLastWord(String s) {
        int i = s.length()- 1 , count = 0;
        while (s.charAt(i) == ' ') {
            i--;
        }

        while (i >= 0 && s.charAt(i) != ' ') {
            count++;
            i--;
        }

        return count;
    }
}

File: Leetcode_Python/String/length_of_last_word.py
def lengthOfLastWord(self, s: str) -> int:
    count , i = 0 ,  len(s) - 1

    while i >= 0 and s[i] == ' ':
        i -= 1

    while i >= 0 and s[i] != ' ':
        count += 1
        i -= 1

    return count